hw/drivers/sensors/bma253/src/bma253.c

```diff
@@ -30,30 +30,56 @@
 #else
 #include "hal/hal_i2c.h"
 #include "i2cn/i2cn.h"
+#include "hal/hal_spi.h"
 #endif
 
+
 #if MYNEWT_VAL(BMA253_LOG)
 #include "modlog/modlog.h"
 #endif
 
+
+#define BMA253_LOG_LEVEL_THRESH LOG_LEVEL_WARN
+
 #if MYNEWT_VAL(BMA253_LOG)
 
 #define BMA253_LOG(lvl_, ...) \
-    MODLOG_ ## lvl_(MYNEWT_VAL(BMA253_LOG_MODULE), __VA_ARGS__)
+    if ((LOG_LEVEL_ ## lvl_) >= BMA253_LOG_LEVEL_THRESH) { MODLOG_ ## lvl_(MYNEWT_VAL(BMA253_LOG_MODULE), __VA_ARGS__); }
```

Comment:
Please do not change this. There is a reason it is that way, we need to be able to have a different log level than `WARN`

Reply:
ok